target/linux/ath79/image/nand.mk

```diff
@@ -1,4 +1,17 @@
 # attention: only zlib compression is allowed for the boot fs
+define Build/kt-headers
+	dd if=$@ of=$@.tmp bs=4 count=1 && \
+	printf '\x00\x00\x00\x00\x00\x02\x04\x00' | dd of=$@.tmp bs=1 seek=4 conv=notrunc && \
+	dd bs=8 skip=1 if=$@ of=$@.tmp oflag=append conv=notrunc && \
```

Comment:
`bs=8` here will slow down the copy speed a lot. The easiest solution is `skip_bytes`, but I think we should avoid using it on host (as macOS dd does not support the option) so I hope there would be a better way...

Reply:
I found found a way to replace this, but I don't know if this method will be faster than the previous one.
`tail -c +9 $@ >> @.tmp`

Reply:
This new method seems to reduce processing time very much.

`# time tail -c +9 1 >> 1.tmp`
`real    0m0.004s`
`user    0m0.004s`
`sys     0m0.000s`

`# time dd bs=8 skip=1 if=1 of=1.tmp oflag=append conv=notrunc`
`real    0m0.590s`
`user    0m0.160s`
`sys     0m0.430s`

Reply:
Wow, that's a way better. Great!